microcosm_pubsub/handlers.py

```diff
@@ -55,11 +56,18 @@ def __call__(self, message):
 
         resource = self.get_resource(message, uri)
 
-        if self.handle(message, uri, resource):
-            self.on_handle(message, uri, resource)
+        extra = dict(
```

Comment:
I'm ok with this, but I think the more correct approach is to inject into the graph's current opaque dictionary.

@cdm-ium can help you with that